host.yaml

```diff
@@ -0,0 +1,89 @@
+ref: "openshift/3.10/x86_64/os"
+repos:
+  - atomic-centos-continuous
+  - rhcos-continuous
+  - openshift
+  - rhel-7.5-server
+  - rhel-7.5-server-optional
+  - rhel-7.5-server-extras
+  - rhel-7.5-atomic
+install_langs:
+ - en_US
+documentation: false
+initramfs-args:
+  - "--no-hostonly"
+  - "--add"
+  - "iscsi"
+automatic_version_prefix: "3.10-7.1803"
+mutate-os-release: "7"
+postprocess-script: "compose-post.sh"
+etc-group-members:
+  - wheel
+  - docker
+ignore-removed-users:
+  - root
+ignore-removed-groups:
+  - root
+check-passwd:
+  type: "file"
+  filename: "passwd"
+check-groups:
+  type: "file"
+  filename: "group"
+packages: 
```

Comment:
I really like the package breakdown like this!

Reply:
One other thing I did in `coreos-assembler` is that not only is it YAML, each line in the `packages` section can be space-separated too.  It feels slightly dirty but since package names can't contain whitespace...